.github/workflows/run-zip-test.yml

```diff
@@ -0,0 +1,36 @@
+name: Run Zip tests
+
+on:
+  push:
+  pull_request:
+    branches-ignore: [master]
+
+jobs:
+  zip:
+    runs-on: '${{ matrix.os }}'
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.9', '3.10', '3.11-dev']
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: 'Set up Python ${{ matrix.python-version }}'
+        uses: actions/setup-python@v2
+        with:
+          python-version: '${{ matrix.python-version }}'
+      - name: Install dependencies
+        run: |
+          python scripts/ci/install
+      - name: Run tests
+        run: |
+          if [ "$RUNNER_OS" == "Windows" ]; then
+            export PYTHONPATH='${{ github.workspace }}\botocore.zip'
+          else
+            export PYTHONPATH=${{ github.workspace }}/botocore.zip
+          fi
+          python -m zipfile -c botocore.zip botocore/
+          python scripts/ci/run-zip-tests --with-cov --with-xdist --append-import
+          rm botocore.zip
+        shell: bash
```

Comment:
this took a number of iterations to figure out for windows. setting an environment variable in different steps via
```bash
echo "PYTHONPATH=${{ github.workspace }}\botocore.zip" >> $GITHUB_ENV
```
worked fine for mac/linux, but consistently had issues with Windows. I think it has to do with how gh actions treats forward slashes as escapes. Passing it between steps would result in it disappearing so I would see something like this
```bash
echo ${{ env.PYTHONPATH}}
D:\a\botocore\botocorebotocore.zip
```

Reply:
i was getting a parsing error with the if statement, but it doesn't appear to be an issue now